File: app/src/main/java/com/apps/adrcotfas/goodtime/Settings/reminders/ReminderReceiver.java
package com.apps.adrcotfas.goodtime.Settings.reminders;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import android.util.Log;

public class ReminderReceiver extends BroadcastReceiver {
    private static final String TAG = "ReminderReceiver";

    @Override
    public void onReceive(Context context, Intent intent) {
        Log.d(TAG, "onReceive");
        ReminderHelper.notifyReminder(context);
        ReminderHelper.scheduleNotification(context);
    }
}

File: app/src/main/java/com/apps/adrcotfas/goodtime/settings/reminders/ReminderReceiver.kt
package com.apps.adrcotfas.goodtime.settings.reminders

import com.apps.adrcotfas.goodtime.settings.reminders.ReminderHelper.Companion.notifyReminder
import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import android.util.Log

class ReminderReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        Log.d(TAG, "onReceive")
        notifyReminder(context)
    }

    companion object {
        private const val TAG = "ReminderReceiver"
    }
}